src/java.base/share/classes/java/lang/String.java

```diff
@@ -689,12 +689,24 @@ private static String decode(Charset charset, byte[] bytes, int offset, int leng
     }
 
     /*
```

Comment:
Is there a reason, why it can't be _converted_ to a proper javadoc?
```suggestion
    /**
```

Reply:
Improved as suggested in 8d7dcb4d410.